tests/testsuite/script.rs

```diff
@@ -30,19 +30,18 @@ fn basic_rs() {
 
     p.cargo("-Zscript -v echo.rs")
         .masquerade_as_nightly_cargo(&["script"])
-        .with_stdout(
-            r#"bin: [..]/debug/echo[EXE]
+        .with_stdout_data(str![[r#"
+bin: [ROOT]/[..]/debug/echo[EXE]
 args: []
-"#,
-        )
-        .with_stderr(
-            "\
+
+"#]])
+        .with_stderr_data(str![[r#"
 [WARNING] `package.edition` is unspecified, defaulting to `2021`
 [COMPILING] echo v0.0.0 ([ROOT]/foo)
-[FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [..]s
-[RUNNING] `[..]/debug/echo[EXE]`
-",
-        )
+[FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+[RUNNING] `[ROOT]/[..]/debug/echo[EXE]`
```

Comment:
How come `echo` needs globbing?

Reply:
We're globbing it because the path looks like
```
[RUNNING] `[ROOT]/home/.cargo/target/a4/7b05b555ee1ab6/debug/echo`
```

Reply:
We have a `[HASH]` redaction.  Could you extend it to include this?  Something like `r".cargo/target/(?<redacted>[0-9a-f]{2}/[0-9a-f]{14})/"`?